tests/bluetooth/tester/src/btp_csip.c

```diff
@@ -189,15 +190,53 @@ static uint8_t btp_csip_discover(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_VAL(err);
 }
 
+int available_members(const struct bt_csip_set_coordinator_set_member **members)
+{
+	members_count = 0;
+
+	if (cur_csis_inst == NULL) {
+		LOG_ERR("No CISP instance available");
+		return 1;
```

Comment:
Using `1` as a return value seems kind of odd. 

Did you mean to use `BTP_STATUS_FAILED`? 

Alternatively you can consider making this a `ssize_t` return value, so you can return the number of members added (rather than relying on a global variable), and return `-1` on error.

Reply:
Went with BTP_STATUS_FAILED/BTP_STATUS_SUCCESS instead of 0 and 1.